muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/guider/GuiderRESTService.java

```diff
@@ -659,6 +668,49 @@ public Response getTranscriptOfRecordsFileContent(@PathParam("ID") Long fileId)
     return Response.ok().type(contentType).entity(output).build();
   }
   
+  /**
+   * POST mApi().guider.students.studyTime.create(16, {months: 3})
+   * 
+   * Increases student's study time end by given months.
+   * 
+   * @param userEntityId
+   * @param months
+   * @return 
+   * 
+   * returns increased studyTimeEnd
+   */
+  
+  @POST
+  @Path("/students/{ID}/studyTime")
+  @RESTPermit (handling = Handling.INLINE, requireLoggedIn = true)
```

Comment:
Where will this endpoint even be used? Only checks that the user is logged in, so basically anyone could call it and add study time for anyone? Access definitely needs to be more limited.